src/smoosh/src/smoosh_channel.erl

```diff
@@ -472,11 +472,19 @@ maybe_start_compaction(State) ->
 start_compact(State, DbName) when is_list(DbName) ->
     start_compact(State, ?l2b(DbName));
 start_compact(State, DbName) when is_binary(DbName) ->
-    {ok, Db} = couch_db:open_int(DbName, []),
-    try
-        start_compact(State, Db)
-    after
-        couch_db:close(Db)
+    case couch_db:open_int(DbName, []) of
+        {ok, Db} ->
+            try
+                start_compact(State, Db)
+            after
+                couch_db:close(Db)
+            end;
+        {not_found, no_db_file} ->
```

Comment:
Minor nit: to avoid repeating the error below can do `Err = {not_found, no_db_file}  ->` then in the warning use `Err`

Reply:
Changed, thanks Nick.